.circleci/config.yml

```diff
@@ -48,8 +48,11 @@ jobs:
         
       # run tests!
       - run: mvn pmd:check
-      - run: mvn integration-test 
-      #- run: mvn cobertura:cobertura          
+      # validate code format 
+      - run: mvn git-code-format:validate-code-format -Dgcf.globPattern=**/*
+      # run tests!
+      - run: mvn integration-test
+      - run: mvn cobertura:cobertura       
```

Comment:
@filipecosta90 why did you put the `mvn cobertura:cobertura` back?

Reply:
by mistake. removed commented line as discussed